### PR TITLE
Feature/send offer from chat

### PR DESCRIPTION
**Pull request by DavidSurkov**

### CODE REVIEW

File: src/pages/Chat/components/ChatRoom/ChatRoom.tsx [hunk 1]
```diff
@@ -42,10 +46,17 @@ function ChatRoom({ userSelfId, userRole }: IChatRoom) {
...
-            </Title>
+            <div>
+              <Title fontSize="16px">{t("Chat.jobTitle")}</Title>
+              <Title italic bold>
+                {room?.jobPost?.title}
+              </Title>
+            </div>
+            {userRole === UserRoleEnum.CLIENT && (
+              <StyledButton shape="round" onClick={toggleModal}>
+                Send offer
```
Comment (xx-TOOL-xx) [inline src/pages/Chat/components/ChatRoom/ChatRoom.tsx]: Localization

File: src/pages/Chat/components/ChatRoom/ChatRoom.tsx [hunk 2]
```diff
@@ -83,18 +94,32 @@ function ChatRoom({ userSelfId, userRole }: IChatRoom) {
...
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+          }}
+        >
+          <div style={{ fontSize: "24px" }}>Send offer</div>
+          <div style={{ fontSize: "18px" }}>
+            Do you want to send the offer to this user?
```
Comment (xx-TOOL-xx) [inline src/pages/Chat/components/ChatRoom/ChatRoom.tsx]: Same

**Review by xx-TOOL-xx — APPROVED** [review]

File: src/pages/Chat/components/ChatRoom/ChatRoom.tsx (hunk 2 above)
Comment (DavidSurkov) [reply src/pages/Chat/components/ChatRoom/ChatRoom.tsx]: fixed

File: src/pages/Chat/components/ChatRoom/ChatRoom.tsx (hunk 1 above)
Comment (DavidSurkov) [reply src/pages/Chat/components/ChatRoom/ChatRoom.tsx]: fixed

**Pull request merged**